tests/test_logging.py

```diff
@@ -0,0 +1,86 @@
+import os
+import sys
+
+
+def _gw_logging_cfg(log_file):
+    return {
+        'version': 1,
+        'disable_existing_loggers': False,
+        'formatters': {
+            'standard': {
+                'format': "%(asctime)s - %(levelname)-5s - %(message)s"
+            },
+            'debug': {
+                'format': "%(asctime)s - %(levelname)-5s - %(name)-40s - %(message)-80s - %(module)s:%("
+                          "funcName)s(%(lineno)s)"
+            },
+        },
+        'handlers': {
+            'console_stdout': {
+                'formatter': 'standard',
+                'class': 'logging.StreamHandler',
+                'stream': sys.stdout,
+                'level': 'INFO'
+            },
+            'file': {
+                "class": "logging.handlers.RotatingFileHandler",
+                "formatter": "debug",
+                "filename": log_file,
+                "maxBytes": 1024000,
+                "backupCount": 3,
+                'level': 'INFO'
+            },
+        },
+        'loggers': {
+            '': {
+                'handlers': ['console_stdout'],
+                'level': 'INFO',
+                'propagate': False
+            },
+            'groundwork': {
+                'handlers': ['console_stdout', 'file'],
+                'level': 'INFO',
+                'propagate': False
+            },
+        }
+    }
+
+
+def test_logging_file(basicApp, tmpdir):
```

Comment:
Some description for the test case would be nice; I know it's not the current state of the project but I have to read the code to get what the purpose of this test case is.